index.bs

```diff
@@ -3701,6 +3727,792 @@ opened</dfn> steps given |window|, |type| and |message|.
 
 </div>
 
+## The network Module ## {#module-network}
+
+The <dfn export for=modules>network</dfn> module contains commands and events
+relating to network requests.
+
+### Definition ### {#module-network-definition}
+
+<pre class="cddl remote-cddl">
+
+NetworkCommand = (
+)
+
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+
+NetworkResult = (
+)
+
+NetworkEvent = (
+    network.BeforeRequestSent //
+    network.ResponseStarted //
+    network.ResponseCompleted //
+    network.FetchError
+)
+
+</pre>
+
+A [=remote end=] has a <dfn>before request sent map</dfn> which is initially an
+empty map.  It's used to track the network events for which a
+<code>network.beforeRequest</code> event has already been sent.
+
+### Types ### {#module-network-types}
+
+#### The network.BaseEventParams type #### {#type-network-BasEventParams}
+
+<pre class="cddl local-cddl">
+network.BaseParameters = {
+    request: network.RequestData,
```

Comment:
Yes, I wondered a bit about that. I think if we did that we'd also want to make `response` into `responseData`, and I think it's a bit different to CDP which has `requestId` and `request`, but we have the precedent that ids are given the name of the thing they're an ID for (e.g. `context` rather than `contextId`).